internal/backend/db/dbgorm/dbgorm.go

```diff
@@ -168,15 +183,31 @@ func (db *gormdb) Setup(ctx context.Context) error {
 func (db *gormdb) Teardown(ctx context.Context) error {
 	isSqlite := db.cfg.Type == "sqlite"
 	if isSqlite {
-		db.db.Exec("PRAGMA foreign_keys = OFF")
+		foreignKeys(db, false)
 	}
 	if err := db.db.WithContext(ctx).Migrator().DropTable(scheme.Tables...); err != nil {
 		return fmt.Errorf("droptable: %w", err)
 	}
 	if isSqlite {
-		db.db.Exec("PRAGMA foreign_keys = ON")
+		foreignKeys(db, true)
+	}
+
+	return nil
+}
+
+func (gormdb *gormdb) Cleanup(ctx context.Context) error {
```

Comment:
I think having this cleanup method as dangerous, someone might call it by mistake on a production database, I think it belongs on the tests code only

same as teardown, we don't really need it 

@itayd / @daabr WDYT ?

Reply:
@efiShtain 
teardown is used as CLI command as well